src/core/main.cpp

```diff
@@ -120,36 +120,36 @@ void printHelp()
 {
 	printf( "LMMS %s\n"
 		"Copyright (c) 2004-%s LMMS developers.\n\n"
-		"usage: lmms [ -r <project file> ] [ options ]\n"
+		"Usage: lmms [ -r <project file> ] [ options ]\n"
```

Comment:
I probably should have checked Linux, I was on a BSD flavor at the time of trying.  It looks better your way anyway. :+1: